packages/typography/src/Typography.story.tsx

```diff
@@ -59,6 +59,7 @@ storiesOf('Typography', module).add('Default', () => {
             'select Link href',
             [
               'http://localhost:9001',
+              '?path=/story/button--icon-only',
```

Comment:
this could be a good opportunity to also update the website

Reply:
Would the website would be updated in a separate PR since it references the published package, not the local update?

Reply:
nope, same PR works... the website references the local packages not the published ones!